### PR TITLE
Update activity-1-instructions.Rmd so Details goes to a description

### PR DESCRIPTION
**Pull request by kweav**
Removed a forward slash from the first <summary> hoping that the description that is there will replace the "Details" that is rendered.

### CODE REVIEW

File: activity-1-instructions.Rmd [hunk 1]
```diff
@@ -98,7 +98,7 @@ Our container is separate from our computer so if we want to use a file we have
 
 Let's point a volume to our workshop files so we have them on our container.
 
-<details> </summary> Other ways we could get files on a container </summary>
+<details> <summary> Other ways we could get files on a container </summary>
```
Comment (carriewright11) [inline activity-1-instructions.Rmd:101]:
```suggestion
<details> <summary> Click here for other ways we could get files on a container. </summary>
```

File: activity-1-instructions.Rmd [hunk 2]
```diff
@@ -209,7 +209,7 @@ Our container is separate from our computer so if we want to use a file we have
...
```
Comment (carriewright11) [inline activity-1-instructions.Rmd:212]:
```suggestion
<details> <summary> Click here for other ways to get files on a container.</summary>
```

**Comment by kweav** [comment]
I like the "Click here"'s a lot!

**Comment by carriewright11** [comment]
> I like the "Click here"'s a lot!

I am going to add some styling to make a box around it too - PR coming!

**Pull request merged**